### PR TITLE
enable translation for tab titles

### PR DESCRIPTION
**Pull request by sardhani-ch**

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by ttingle-ch — APPROVED** [review]
LGTM

File: src/main/controllers/features/common/applicationConfirmationController.ts [hunk 1]
```diff
@@ -18,7 +18,7 @@ export const get = async (req: Request, res: Response, next: NextFunction) => {
         const transactionId: string = session.getExtraData(SUBMISSION_ID)!;
 
         res.render(config.APPLICATION_CONFIRMATION, {
-            title: "Application submitted",
+            title: locales.i18nCh.resolveNamespacesKeys(lang).applicationSubmittedTitle,
```
Comment (swalke-ch) [inline src/main/controllers/features/common/applicationConfirmationController.ts:21]: I think you don't need to do resolveNamespacesKeys(lang) here. getLocaleInfo in net line doing same thing.

**Pull request merged**